ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/CliCommand.java

```diff
@@ -79,4 +83,16 @@ public String getLongHelp(final String command) {
         return (View<T>) new DefaultView();
     }
 
+    public static String getPlatformInformation() {
+        StringBuilder ret = new StringBuilder();
```

Comment:
I'm actually surprised that intellij is not complaining about this saying that it can refactored to string concatenation